src/Microsoft.AspNet.OData.Shared/Query/Expressions/PropertyContainer.generated.cs

```diff
@@ -1,4 +1,11 @@
-﻿// <auto-generated />
+//-----------------------------------------------------------------------------
+// <copyright file="PropertyContainer.generated.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved. 
+//      See License.txt in the project root for license information.
+// </copyright>
+//------------------------------------------------------------------------------
+
+// <auto-generated />
```

Comment:
Verify if this file is auto-generated. If true then I'd be curious if we need to add the copyright header

Reply:
@marabooy You can ignore this comment. I think it's right for this file to have the copyright header. It just happens that it was auto-generated somewhere else before being added to this project